tests/test.py

```diff
@@ -657,7 +657,12 @@ def test_clearing(self):
         # execute any query for loading key by acra
         result = self.engine1.execute(sa.select([1]).limit(1))
         result.fetchone()
-        with urlopen('http://localhost:{}/resetKeyStorage'.format(self.ACRASERVER_PORT + 1)) as response:
+        import ssl
```

Comment:
lets move this import to the top of the file where all imports should be)

and lets add `check_hostname` for SSL to be stricter as we do it for asyncpg executor - https://github.com/cossacklabs/acra/blob/master/tests/base.py#L1198